packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -545,6 +547,29 @@ export const PersonalDetailsForm = ({
         if (!form_initial_values.employment_status) form_initial_values.employment_status = '';
     }
 
+    const is_verified =
+        authentication_status?.document_status === 'verified' && authentication_status?.identity_status === 'verified';
+
+    const getRedirectionLink = () => {
+        if (
+            authentication_status?.document_status !== 'verified' &&
+            authentication_status?.identity_status !== 'verified'
+        ) {
+            return '/account/proof-of-identity';
+        } else if (
+            authentication_status?.document_status === 'verified' &&
+            authentication_status?.identity_status !== 'verified'
+        ) {
+            return '/account/proof-of-identity';
+        } else if (
+            authentication_status?.document_status !== 'verified' &&
+            authentication_status?.identity_status === 'verified'
+        ) {
+            return '/account/proof-of-address';
+        }
+        return null;
+    };
```

Comment:
- Very hard to read, you can leave some comments.
- Try to simplify the function.. use copilot.